test/ufuzz.js

```diff
@@ -573,6 +593,10 @@ function createTypeofExpr() {
         return 'typeof ' + createVarName(MANDATORY, DONT_STORE) + ' != "' + TYPEOF_OUTCOMES[rng(TYPEOF_OUTCOMES.length)] + '"';
       case 4:
         return 'typeof ' + createVarName(MANDATORY, DONT_STORE);
+      case 5:
+      case 6:
+      case 7:
```

Comment:
It's not worth it. Too rare in the wild. Would slow down Compressor for little gain.